examples/06-custom-schema/react-custom-blocks/src/App.tsx

```diff
@@ -41,8 +41,8 @@ export const alertBlock = createReactBlockSpec(
       textAlignment: defaultProps.textAlignment,
       textColor: defaultProps.textColor,
       type: {
-        default: "warning",
-        values: ["warning", "error", "info", "success"],
+        default: "warning" as const,
```

Comment:
is the `as const` necessary now where before it wasn't? it's not ideal imo (the `as const` syntax is probably not understood by basic TS devs, so if we can avoid it that'd be better ofc)